app/lib/frontend/templates/package_admin.dart

```diff
@@ -18,17 +18,20 @@ String renderPkgAdminPage(
   PackagePageData data,
   List<String> userPublishers,
   List<User> uploaderUsers,
+  List<String> retractableVersions,
+  List<String> retractedVersions,
 ) {
   final tabs = buildPackageTabs(
     data: data,
     adminTab: Tab.withContent(
       id: 'admin',
       title: 'Admin',
       contentNode: packageAdminPageNode(
-        package: data.package!,
-        userPublishers: userPublishers,
-        uploaderUsers: uploaderUsers,
-      ),
+          package: data.package!,
+          userPublishers: userPublishers,
+          uploaderUsers: uploaderUsers,
+          retractableVersions: retractableVersions,
+          retractedVersions: retractedVersions),
```

Comment:
nit: trailing comma makes for a nicer diff :D
```suggestion
          retractedVersions: retractedVersions,),
```

Reply:
done